src/main/java/gameframework/drawing/GameUniverseViewPortDefaultImpl.java

```diff
@@ -10,11 +10,16 @@
 import java.util.Iterator;
 
 public class GameUniverseViewPortDefaultImpl implements GameUniverseViewPort {
+
 	private Image buffer;
 	protected BackgroundImage background;
 	protected GameData data;
 
-	protected URL backgroundImage() {
+	protected URL backgroundImage(String path) {
+		return this.getClass().getResource(path);
+	}
+
+	protected URL backgroundImage() { // with default pathname
 		return this.getClass().getResource("/images/background_image.gif");
```

Comment:
please call method `backgroundImage(String)` instead of copy/pasting content. This will have the added benefit that subclasses of `GameUniverseViewPortDefaultImpl` will be able to change the implementation of `backgroundImage(String)` and get both `backgroundImage()` and `backgroundImage(String)` affected.